### PR TITLE
Basic healthcheck page

### PR DESCRIPTION
**Pull request by h-lame**
For: https://www.pivotaltracker.com/story/show/95514606

### CODE REVIEW

File: lib/health_check_middleware.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+require 'health_check'
+
+class HealthCheckMiddleware
+  def initialize(app)
+    @app = app
+  end
+
+  def call(env)
+    if env['PATH_INFO'] =~ %r[/health-check/?\Z]
```
Comment (pixeltrix) [inline lib/health_check_middleware.rb]: Do we want to anchor this to the beginning of the string as well?

**Comment by pixeltrix** [comment]
I'm worried that some of those Socket methods could be time-consuming - especially ones relating to DNS lookups and that it could end up triggering an autoscale event on the load balancer when it's pinging app servers for status.

Also, I wonder whether we should be returning 5xx errors if things like the DB connection isn't active?

**Comment by alexander-bobin** [comment]
This PR is stagnating. Are we happy to leave it for when Murray gets back?

**Comment by yahoopete** [comment]
@h-lame is back Monday, so we can leave it till then.

**Comment by pixeltrix** [comment]
@h-lame did you see my note about the path regex? It'll match any url ending with `/health-check` currently - is this intentional?

**Comment by h-lame** [comment]
Agh, no. I missed it.  Will fix and reupload.

**Pull request merged**